src/components/Property/RaiseWorkOrder/RaiseWorkOrderFormView.js

```diff
@@ -136,6 +136,9 @@ const RaiseWorkOrderFormView = ({ propertyReference }) => {
       const data = await frontEndApiRequest({
         method: 'get',
         path: `/api/properties/${propertyReference}`,
+        params: {
+          withContactDetails: true,
+        },
       })
```

Comment:
Could there be any other places in RH where the `frontEndApiRequest` needs to be edited or is this the only instance where we do require contact details?

Reply:
Nope. Only when raising a repair. The contact details information is provided as a suggestion when filling out the workOrder contact details. After the repair is raised, contact information is stored in repairs hub directly